scripts/install-nix-from-closure.sh

```diff
@@ -65,6 +65,21 @@ elif [ "x${1:-}" != "x" ]; then
     exit
 fi
 
+if [ "$(uname -s)" = "Darwin" ]; then
+    canondest="/usr/local/nix"
```

Comment:
I thought we wanted to use `/opt`, not `/usr/local`?

Reply:
Either is find by me, I went for this because it's next to homebrew.

Reply:
Homebrew is trying to own the entire `/usr/local` tree (which is a poor decision but one they made so existing software would automatically pick up Homebrew packages without touching the `PATH`). `/opt` is the proper location for third-party package managers to use.

Reply:
+1 on using `/opt/nix` instead of using `/usr/local` to keep away from Homebrew.

Reply:
Also +1 on `/opt/nix`.

Reply:
Updated, since it seems like there's a consensus.